### PR TITLE
Update top consumers description and units

### PR DESCRIPTION
**Pull request by yaacov**
We want to update the top consumers data collected in web-ui, this PR update the description and units to match this cahnges.

Depends on kubevirt/web-ui#351

Ref:
https://jira.coreos.com/browse/CNV-1157

### CODE REVIEW

**Comment by yaacov** [comment]
Waiting for kubevirt/web-ui#351 to finalize units and strings.

**Comment by yaacov** [comment]
Removing wip,
Shoule be merged near the same time as https://github.com/kubevirt/web-ui/pull/351

File: src/components/ClusterOverview/TopConsumers/tests/TopConsumers.test.js [hunk 1]
```diff
@@ -1,14 +1,98 @@
...
+
 describe('<TopConsumers />', () => {
-  it('renders correctly', () => {
+  it('renders correctly (PODS_AND_VMS BY_CPU)', () => {
     const component = render(testTopConsumersOverview());
     expect(component).toMatchSnapshot();
   });
+
+  it('renders correctly (PODS_AND_VMS BY_MEMORY)', () => {
+    const component = mount(testTopConsumersOverview());
```
Comment (rawagner) [inline src/components/ClusterOverview/TopConsumers/tests/TopConsumers.test.js]: mounting this card results in a big snapshots which are hard to follow for future updates. I'd suggest to remove the test updates from this PR and use https://github.com/kubevirt/web-ui-components/pull/432/files#diff-f866a1899292fa89cfdd5c4b02bfb306R39

**Comment by yaacov** [comment]
> I'd suggest to remove the test updates from this PR and use https://github.com/kubevirt/web-ui-components/pull/432/files#diff-f866a1899292fa89cfdd5c4b02bfb306R39

@rawagner thanks,  :+1: 

Remove tests from this one, in favor of https://github.com/kubevirt/web-ui-components/pull/432
